pkg/api/user_token.go

```diff
@@ -224,6 +225,12 @@ func (hs *HTTPServer) getUserAuthTokensInternal(c *contextmodel.ReqContext, user
 			seenAt = createdAt
 		}
 
+		// Retrieve AuthModule from external session
+		authModule := ""
```

Comment:
Maybe we should set a default like `internal` and show that label the user is internal and not authenticated by an external IdP. Wdyt?

Reply:
I thought about it and I think leaving it empty would make more sense.